modules/shadertools/src/modules/lighting/lights/lighting-uniforms-wgsl.ts

```diff
@@ -0,0 +1,57 @@
+// luma.gl
+// SPDX-License-Identifier: MIT
+// Copyright (c) vis.gl contributors
+
+export const lightingUniformsWGSL = /* wgsl */ `\
+// #if (defined(SHADER_TYPE_FRAGMENT) && defined(LIGHTING_FRAGMENT)) || (defined(SHADER_TYPE_VERTEX) && defined(LIGHTING_VERTEX))
+struct AmbientLight {
+  color: vec3<f32>,
+};
+
+struct PointLight {
+  color: vec3<f32>,
+  position: vec3<f32>,
+  attenuation: vec3<f32>, // 2nd order x:Constant-y:Linear-z:Exponential
+};
+
+struct DirectionalLight {
+  color: vec3<f32>,
+  direction: vec3<f32>,
+};
+
+struct lightingUniforms {
+  enabled: i32,
+  poightCount: i32,
```

Comment:
```suggestion
  pointLightCount: i32,
```

Reply:
Yes, I renamed in [similar situations](https://github.com/visgl/deck.gl/pull/9001/files#diff-73faf4dc6652b8783778e5811d4d092b37fbe530573918b0505db1fb616f8d0fL48) when porting in deck